dts/bindings/gpio/gpio-leds.yaml

```diff
@@ -18,3 +26,7 @@ child-binding:
           required: true
           type: string
           description: Human readable string describing the device (used by Zephyr for API name)
+       id:
+          required: false
+          type: int
+          description: Id needed for GPIO LED driver to identify LED
```

Comment:
`ID needed for GPIO LED driver to identify an LED`

Reply:
Changed to` ID needed for the GPIO LED driver to identify an LED`